engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -666,6 +671,80 @@ class RuntimeServerTest
     context.consumeOut shouldEqual List("79")
   }
 
+  it should "send method pointer updates of constructors" in {
+    val contextId  = UUID.randomUUID()
+    val requestId  = UUID.randomUUID()
+    val moduleName = "Enso_Test.Test.Main"
+
+    val metadata = new Metadata
+    val idA      = metadata.addItem(47, 3, "aa")
+    val idB      = metadata.addItem(59, 6, "ab")
+    val idC      = metadata.addItem(70, 7, "ac")
+
+    val code =
+      """type T
+        |    A
+        |    B x
+        |    C y z
+        |
+        |main =
+        |    a = T.A
+        |    b = T.B 42
+        |    T.C a b
+        |""".stripMargin.linesIterator.mkString("\n")
+    val contents = metadata.appendToCode(code)
+    val mainFile = context.writeMain(contents)
+
+    // create context
+    context.send(Api.Request(requestId, Api.CreateContextRequest(contextId)))
+    context.receive shouldEqual Some(
+      Api.Response(requestId, Api.CreateContextResponse(contextId))
+    )
+
+    // open file
+    context.send(
+      Api.Request(Api.OpenFileNotification(mainFile, contents))
+    )
+    context.receiveNone shouldEqual None
+
+    // push main
+    context.send(
+      Api.Request(
+        requestId,
+        Api.PushContextRequest(
+          contextId,
+          Api.StackItem.ExplicitCall(
+            Api.MethodPointer(moduleName, moduleName, "main"),
+            None,
+            Vector()
+          )
+        )
+      )
+    )
+    context.receiveN(5) should contain theSameElementsAs Seq(
+      Api.Response(requestId, Api.PushContextResponse(contextId)),
+      TestMessages.update(
+        contextId,
+        idA,
+        "Enso_Test.Test.Main.T",
+        Api.MethodPointer("Enso_Test.Test.Main", "Enso_Test.Test.Main.T", "A")
+      ),
+      TestMessages.update(
+        contextId,
+        idB,
+        "Enso_Test.Test.Main.T",
+        Api.MethodPointer("Enso_Test.Test.Main", "Enso_Test.Test.Main.T", "B")
+      ),
+      TestMessages.update(
+        contextId,
+        idC,
+        "Enso_Test.Test.Main.T",
```

Comment:
Looks like the type is duplicated for constructors. First of all it is specified as `expressionType` and then it is repeated in the method pointer.

It is probably not a huge problem, but there is some potential to decrease the size of the messages in the future.